### PR TITLE
Auto plugin docs.

### PR DESCRIPTION
**Pull request by rasapala**

### CODE REVIEW

File: docs/accelerators.md [hunk 1]
```diff
@@ -159,3 +159,22 @@ Here is a config example using heterogeneous plugin with GPU as the primary devi
...
+
+   .. code-block:: sh
+
+        docker run --rm -d -v <model_path>:/opt/model -p 9001:9001 openvino/model_server:latest \
+            --model_path /opt/model --model_name my_model --port 9001 \
+            --target_device AUTO
+
+@endsphinxdirective
+
+> NOTE: --target_device AUTO and --shape 'auto' should not be used together.
```
Comment (dtrawins) [inline docs/accelerators.md:180]:
```suggestion
> NOTE: currently, AUTO plugin cannot be used with `--shape auto` parameter while GPU device is enabled. 
```

File: docs/accelerators.md [hunk 2]
```diff
@@ -159,3 +159,22 @@ Here is a config example using heterogeneous plugin with GPU as the primary devi
...
+[Auto Device](https://docs.openvino.ai/nightly/openvino_docs_IE_DG_supported_plugins_AUTO.html) (or AUTO in short) is a new special “virtual” or “proxy” device in the OpenVINO toolkit, it doesn’t bind to a specific type of HW device.
+AUTO solves the complexity in application required to code a logic for the HW device selection (through HW devices) and then, on the deducing the best optimization settings on that device.
+AUTO always chooses the best device, if compiling model fails on this device, AUTO will try to compile it on next best device until one of them succeeds.
+
+To enable AUTO Plugin as target device, use the following command:
+@sphinxdirective
+
+   .. code-block:: sh
+
+        docker run --rm -d -v <model_path>:/opt/model -p 9001:9001 openvino/model_server:latest \
```
Comment (dtrawins) [inline docs/accelerators.md:174]: such command will work only with CPU device. While other device is the be enabled, also it's special docker parameters needs to be added also.
For example with GPU also --device /dev/dri and group context etc.
It should be mentioned are even added to the example

File: docs/accelerators.md [hunk 3]
```diff
@@ -159,3 +159,22 @@ Here is a config example using heterogeneous plugin with GPU as the primary devi
...
 }
 ```
+
+## Using AUTO Plugin
+
+[Auto Device](https://docs.openvino.ai/nightly/openvino_docs_IE_DG_supported_plugins_AUTO.html) (or AUTO in short) is a new special “virtual” or “proxy” device in the OpenVINO toolkit, it doesn’t bind to a specific type of HW device.
+AUTO solves the complexity in application required to code a logic for the HW device selection (through HW devices) and then, on the deducing the best optimization settings on that device.
+AUTO always chooses the best device, if compiling model fails on this device, AUTO will try to compile it on next best device until one of them succeeds.
+
+To enable AUTO Plugin as target device, use the following command:
```
Comment (dtrawins) [inline docs/accelerators.md:169]: It should be mentioned how to tune the performance with AUTO plugin - PERFORMANCE_HINT LATENCY or THROUGHPUT. What is the default?

**Review by dtrawins — CHANGES_REQUESTED** [review]

File: docs/accelerators.md [hunk 4]
```diff
@@ -159,3 +158,61 @@ Here is a config example using heterogeneous plugin with GPU as the primary devi
...
+
+## Using AUTO Plugin
+
+[Auto Device](https://docs.openvino.ai/nightly/openvino_docs_IE_DG_supported_plugins_AUTO.html) (or AUTO in short) is a new special “virtual” or “proxy” device in the OpenVINO toolkit, it doesn’t bind to a specific type of HW device.
+AUTO solves the complexity in application required to code a logic for the HW device selection (through HW devices) and then, on the deducing the best optimization settings on that device.
+AUTO always chooses the best device, if compiling model fails on this device, AUTO will try to compile it on next best device until one of them succeeds.
+Make sure you have passed the devices and access to the devices you want to use in for the docker image. For example with:
+```--device=/dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) -u $(id -u):$(id -g)```
+
+To enable AUTO Plugin as target device, use the following command:
```
Comment (dtrawins) [inline docs/accelerators.md:170]:
```suggestion
Below is an example of the command with AUTO Plugin as target device. It includes extra docker parameters to enable GPU (/dev/dri) , beside CPU.
```

File: docs/accelerators.md [hunk 5]
```diff
@@ -159,3 +158,61 @@ Here is a config example using heterogeneous plugin with GPU as the primary devi
...
+
+   .. code-block:: sh
+
+        docker run --rm -d --device=/dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) -u $(id -u):$(id -g) -v <model_path>:/opt/model -p 9001:9001 openvino/model_server:latest \
+            --model_path /opt/model --model_name my_model --port 9001 \
+            --target_device AUTO
+
+@endsphinxdirective
+
+## Performance Hints
```
Comment (dtrawins) [inline docs/accelerators.md:181]: It duplicates the content from performance_tuning doc

**Review by dtrawins — APPROVED** [review]

**Review by dkalinowski — APPROVED** [review]

**Pull request merged**